### DISCUSSION TITLE
PT Reports Issue - Demand Register Report and PT Adoption Report

### DISCUSSION THREAD

**Discussion by sachin1302** [upyog/UPYOG#357] [General]
PT Demand Register Report - Acc. to Demand Register whenever new property is created with with Demand entry the same need to reflect in the Demand Register Report according to the logic added and it is done with the help of Scheduler Service(it runs on daily basis which push the data from persisted to reporting service) - as of now it is not working properly.
Similarly ULBs with large Properties Data it is difficult to fetch the data from the report service due to which ULBs are not able to get the Demand Register data accordingly.
PT Adoption Report - We are not able to fetch the data in Adoption Report as we need to increase the query timeout time for this and optimize the Report accordingly.

**Comment by support-upyog** [comment]
A detailed session has been given on this in the call dated 25th Sep.
